psydac/ddm/cart.py

```diff
@@ -72,13 +72,13 @@ def __init__( self, npts, pads, periods, reorder, comm=MPI.COMM_WORLD, multiplic
         assert isinstance( reorder, bool )
         assert isinstance( comm, MPI.Comm )
 
-        multiplicity = tuple(multiplicity) if multiplicity else (1,)*len(npts)
+        shifts = tuple(shifts) if shifts else (1,)*len(npts)
 
         # Store input arguments
         self._npts         = tuple( npts    )
         self._pads         = tuple( pads    )
         self._periods      = tuple( periods )
-        self._multiplicity = multiplicity
+        self._shifts = shifts
         self._reorder      = reorder
         self._comm         = comm
```

Comment:
Please align the equal signs, as it improves readability.

Specifically it allows developers and reviewers to scroll through the code much faster.

Reply:
done